ui/src/app/views/workspace-edit/component.tsx

```diff
@@ -449,78 +532,189 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
+                    onChange={v => this.updateResearchPurpose('anticipatedFindings', v)}
+                    style={{marginTop: '0.5rem'}}/>
+        </WorkspaceEditSection>
+        <WorkspaceEditSection required header={researchPurposeQuestions[4].header}>
+          <Link onClick={() => this.setState({showUnderservedPopulationDetails:
+              !this.state.showUnderservedPopulationDetails})} style={{marginTop: '0.5rem'}}>
+            More info on underserved populations
+            {this.state.showUnderservedPopulationDetails ? <ClrIcon shape='caret' dir='up'/> :
+              <ClrIcon shape='caret' dir='down'/>}
+          </Link>
+          {this.state.showUnderservedPopulationDetails && <div style={styles.text}>
+            A primary mission of the AoU Research Program is to include research participants
+            who are medically underserved or are historically underrepresented in Biomedical
+            Research, or who, because of systematic social disadvantage, experience health
+            disparities.  As a way to assess the research being conducted with a focus on these
+            populations, AoU requires that you indicate the demographic categories you intend
+            to focus your analysis on.
+          </div>}
+          <div style={{marginTop: '0.5rem'}}>
+            <RadioButton name='population' style={{marginRight: '0.5rem'}}
+              onChange={v => this.updateResearchPurpose('population', true)}
+              checked={this.state.workspace.researchPurpose.population}/>
+            <label style={styles.text}>Yes, I am interested in the focused study of specific
+            population(s), either on their own or in comparison to other groups.</label>
+          </div>
+          <div>
+            <RadioButton name='population' style={{marginRight: '0.5rem'}}
+                         onChange={v => this.updateResearchPurpose('population', false)}
+                         checked={!this.state.workspace.researchPurpose.population}/>
+            <label style={styles.text}>No, I am not interested in focusing on
+              specific population(s) in my research.</label>
+          </div>
+          <div style={{...styles.text, marginLeft: '2rem'}}>
+            <strong>If "Yes": </strong> Please specify the demographic category or categories of the
+            population(s) that you are interested in exploring in your study.
+            Select as many as applicable.
+            <div style={{display: 'flex', flexDirection: 'row', flex: '1 1 0',
+              marginTop: '0.5rem'}}>
+              <div style={{display: 'flex', flexDirection: 'column'}}>
+                <LabeledCheckBox label='Race/Ethnicity'
+                  value={this.specificPopulationSelected(SpecificPopulationEnum.RACEETHNICITY)}
+                  onChange={v =>
+                     this.updateSpecificPopulation(SpecificPopulationEnum.RACEETHNICITY, v)}
```

Comment:
Opt nit: Because this is the same enum as value uses, there's a bit of code smell around duplication. Would it be possible to reduce that with variable or map? (Using an array and then mapping over it may give a bit more documentation of what is on which side also.)

Reply:
yeah i think this is a good place to use mapping.  i'm not sold on doing it above.  although even here there is some weird ordering with the two columns, and how "other" needs to go last